src/main/java/com/google/alpollo/model/SongEntity.java

```diff
@@ -17,7 +17,7 @@
    * All types can be seen here: 
    * https://cloud.google.com/natural-language/docs/reference/rest/v1/Entity#type
    */ 
-  private final String type;
+  private String type;
```

Comment:
It's better to make Set of strings since you can have multiple types for one entity, instead of concatenating all types to one string and make it not flexible for usage